src/PIL/features.py

```diff
@@ -128,6 +128,7 @@ def get_supported_codecs() -> list[str]:
     "libjpeg_turbo": ("PIL._imaging", "HAVE_LIBJPEGTURBO", "libjpeg_turbo_version"),
     "libimagequant": ("PIL._imaging", "HAVE_LIBIMAGEQUANT", "imagequant_version"),
     "xcb": ("PIL._imaging", "HAVE_XCB", None),
+    "acceleration": ("PIL._imaging", "acceleration", "acceleration"),
```

Comment:
I think this violates the type hint for `check_feature` by returning a string instead of a bool.

Reply:
I've created https://github.com/uploadcare/pillow-simd/pull/140 to try and address this.

Reply:
@radarhere Thanks!